project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16-M1
```

Comment:
@leonardehrenfried I merged sbt-cross-building in 0.13.16-M1. https://github.com/sbt/sbt/releases/tag/v0.13.16-M1, so you can do:

```
> ^^1.0.0-M5
> ++2.12.2
> compile
```

and that should cross build to sbt 1.0.0-M5.